### PR TITLE
Removes SASS

### PR DESCRIPTION
**Pull request by antonjb**
Removes Node-SASS and associated SASS files, cleans next config, removes
bootstrap css files and a few small refactors.

### CODE REVIEW

**Comment by robdmoore** [comment]
🙌🙌🙌

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "license": "N/A",
   "private": true,
   "scripts": {
-    "dev": "cross-env TESTING_MODE=true APPINSIGHTS_INSTRUMENTATIONKEY=2f1d6a9c-1b88-4f3d-bba1-64b11ffd2362 GET_SUBMISSIONS_URL=https://dddperth-functions-test.azurewebsites.net/api/GetSubmissions SUBMIT_VOTE_URL=https://dddperth-functions-test.azurewebsites.net/api/SubmitVote GET_AGENDA_URL=https://dddperth-functions-test.azurewebsites.net/api/GetAgenda SUBMIT_FEEDBACK_URL=https://dddperth-functions-test.azurewebsites.net/api/SubmitFeedback NODE_OPTIONS=--inspect next",
```
Comment (robdmoore) [inline package.json]: Why did you remove the `--inspect` that allowed us to do breakpoint debugging in vscode?

File: components/global/meta.tsx [hunk 2]
```diff
@@ -51,21 +49,7 @@ const Meta: StatelessComponent<MetaArgs> = ({
...
-        integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
-        crossOrigin="anonymous"
-      />
-      <link
-        rel="stylesheet"
-        href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
-        integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN"
-        crossOrigin="anonymous"
-      />
-      <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" />
```
Comment (robdmoore) [inline components/global/meta.tsx]: Assume we weren't using these fonts?

File: next.config.js [hunk 3]
```diff
@@ -1,10 +1,7 @@
 const withBundleAnalyzer = require('@next/bundle-analyzer')({
   enabled: process.env.ANALYZE === 'true',
 })
-const withSass = require('@zeit/next-sass')
```
Comment (robdmoore) [inline next.config.js]: Should this package also be removed from package.json`?

File: package.json (hunk 1 above)
Comment (antonjb) [reply package.json]: Since the version updates, something is wrong here.

If you add inspect how it's described in the Next CLI documentation you get the following error:
```Error: Use env variable NODE_OPTIONS instead: NODE_OPTIONS="--inspect" next dev```

If you follow the fix instructions, it works, but the next server will randomly crash while you're developing. It's very frustrating. I looked into the issue and it's to do with the fact that multiple node debuggers are trying to launch (but only sometimes). I think the option is from node and not Next.

My hunch is that between cross-env and next and version updates something has changed and it's not working. I imagine if I ran next without cross-env it would be ok, but we need those environment variables. Decided to remove because the crashing was the worst scenario but would like to return the functionality.

File: components/global/meta.tsx (hunk 2 above)
Comment (antonjb) [reply components/global/meta.tsx]: Not in the new design.

File: next.config.js (hunk 3 above)
Comment (antonjb) [reply next.config.js]: Yeah, it has been removed
https://github.com/dddwa/dddperth-website/pull/264/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L51
